qa/tasks/mgr/dashboard/test_health.py

```diff
@@ -254,7 +254,7 @@ def test_full_health(self):
                 'state': str,
                 # @TODO: What type should be expected here?
                 'sync_provider': JList(JAny(none=True))
-            }),
+            }, allow_unknown=True),
```

Comment:
Would it be better to update the schema explicitly?

```
diff --git a/qa/tasks/mgr/dashboard/test_health.py b/qa/tasks/mgr/dashboard/test_health.py
index 9787ea45ff..7efd7b80db 100644
--- a/qa/tasks/mgr/dashboard/test_health.py
+++ b/qa/tasks/mgr/dashboard/test_health.py
@@ -253,7 +253,8 @@ class HealthTest(DashboardTestCase):
                 'rank': int,
                 'state': str,
                 # @TODO: What type should be expected here?
-                'sync_provider': JList(JAny(none=True))
+                'sync_provider': JList(JAny(none=True)),
+                'stretch_mode': bool
             }),
             'osd_map': JObj({
                 # @TODO: define schema for crush map and osd_metadata, among
```

Reply:
@bk201 Sure. The  initial intention was to deliver a quick patch to ensure that no more issues were to fix additionally.
As all the tests passed, now it's done.